userspace/libsinsp/eventformatter.cpp

```diff
@@ -199,7 +199,7 @@ void sinsp_evt_formatter::set_format(output_format of, const std::string& fmt)
 					j++;
 					msize++; // count ')'
 				}
-				
+
```

Comment:
Dang it, I didn't mean to commit whitespace changes 🤦🏻‍♂️ 

We really need to get formatting checked by CI.

Reply:
I agree.

Anyway, considering the affected file name, the change is on topic 😜